search/domain/service.go

```diff
@@ -102,6 +103,14 @@ type (
 		AdditionalAttributes map[string]string
 	}
 
+	// Promotion result during search
+	Promotion struct {
+		Title   string
+		Content string
+		URL     string
+		// Media []string
```

Comment:
Maybe we use the Media Struct from the product domain model here. And we could also add some `AdditionalAttributes map[string]string` to easily add more meta data if needed